synapse/models/infotech.py

```diff
@@ -358,6 +397,129 @@ def getModelDefs(self):
+                        'doc': 'The GECOS field for the POSIX account.',
+                    }),
+                    ('posix:home', ('file:path', {}), {
+                        'doc': "The path to the POSIX account's home directory.",
+                        'ex': '/home/visi',
+                    }),
+                    ('posix:shell', ('file:path', {}), {
+                        'doc': "The path to the POSIX account's default shell.",
+                        'ex': '/bin/bash',
+                    }),
+                    ('windows:sid', ('it:os:windows:sid', {}), {
+                        'doc': 'The Microsoft Windows Security Identifier of the account.',
+                    }),
+                    ('groups', ('array', {'type': 'it:group'}), {
+                        'doc': 'An array of groups that the account is a member of.',
+                    }),
+                )),
+                ('it:group', {}, (
+                    ('name', ('str', {'lower': True, 'strip': True, 'onespace': True}), {
+                        'doc': 'The name of the group.',
+                    }),
+                    ('desc', ('str', {}), {
+                        'doc': 'A brief description of the group.',
+                    }),
+                    ('host', ('it:host', {}), {
+                        'doc': 'The host where the group is registered.',
+                    }),
+                    ('domain', ('it:domain', {}), {
+                        'doc': 'The authentication domain where the group is registered.',
+                    }),
+                    ('groups', ('array', {'type': 'it:group'}), {
+                        'doc': 'Groups that are a member of this group.',
+                    }),
+                    ('posix:gid', ('int', {}), {
+                        'doc': 'The primary group ID of the account.',
+                        'ex': '1001',
+                    }),
+                    ('windows:sid', ('it:os:windows:sid', {}), {
+                        'doc': 'The Microsoft Windows Security Identifier of the group.',
+                    }),
+                )),
+                ('it:logon', {}, (
+                    ('time', ('time', {}), {
+                        'doc': 'The time the logon occured.',
+                    }),
+                    ('success', ('bool', {}), {
+                        'doc': 'Set to false to indicate an unsuccessful logon attempt.',
+                    }),
+                    ('logoff:time', ('time', {}), {
+                        'doc': 'The time the logon session ended.',
+                    }),
+                    ('host', ('it:host', {}), {
+                        'doc': 'The host that the account logged in to.',
+                    }),
+                    ('account', ('it:account', {}), {
+                        'doc': 'The account that logged in.',
+                    }),
+                    ('creds', ('auth:creds', {}), {
```

Comment:
May want to discuss to niddle details...here an it:logon is a first class citizen, represented as an event that has associated account, credentials, host, etc. Contrast with auth:creds, which is an object that associates an identifier (user, email...but NOT it:account), a token (password, password hash) and a target (host, website, etc). Should it:account be added to auth:creds as an identifier (and it:account / it:host be removed from it:logon), so that an it:logon is simply associates a time / status / duration and a few other details with a set of auth:creds?